docs/articles_en/documentation/openvino_ir/operation_sets/operations_specifications/matrix/Inverse_14.rst

```diff
@@ -0,0 +1,83 @@
+.. {#openvino_docs_ops_matrix_Inverse_14}
+
+Inverse
+=======
+
+
+.. meta::
+  :description: Learn about Inverse-14 - a matrix operation that computes the inverse or adjoint of one or multiple input matrices.
+
+**Versioned name**: *Inverse-14*
+
+**Category**: *Matrix*
+
+**Short description**: *Inverse* operation computes either the inverse or adjoint (conjugate transposes) of one or more square invertible matrices.
+
+**Detailed description**: *Inverse* operation computes the inverse of a square matrix. The operation uses LU decomposition with partial pivoting to compute the inverses.
```

Comment:
Why LU decomposition algorithm was chosen? Are there real models with Invert operations and from which framework are they taken? Can it affect accuracy of the PyTorch models?

Reply:
It guarantees some degree of numerical stability while keeping high performance, It is used by both [Tensorflow ](https://www.tensorflow.org/api_docs/python/tf/linalg/inv
)and 
[PyTorch](https://pytorch.org/docs/stable/generated/torch.linalg.inv.html#torch.linalg.inv)